dashboard/test/ui/features/sharepageLogo.feature

```diff
@@ -1,5 +1,5 @@
-@no_mobile
 @as_student
+@no_mobile
```

Comment:
I had implemented these at some point, but I couldn't figure out how to indicate that they were 'only-mobile'. I'll ask around and see if I can figure that out. Thanks!